packages/commonwealth/server/controllers/server_proposal_methods/compound/proposals.ts

```diff
@@ -166,6 +175,54 @@ async function getProposalDataSequentially(
   return [proposalCreatedEvents, proposals, proposalStates];
 }
 
+async function getProposalDataFieldsSequentially(
+  contract: GovernorCountingSimple,
+): Promise<ResolvedProposalPromises> {
+  console.log('Fetching proposal data fields sequentially');
+  const proposalCreatedEvents = await getProposalCreatedEvents(contract);
+
+  const proposalDataPromises: Promise<CompoundProposalType>[] = [];
+  const proposalStatePromises: Promise<number>[] = [];
+  for (const propCreatedEvent of proposalCreatedEvents) {
```

Comment:
What does the event data look like / contain? Wondering if we need to do all of these queries for every proposal all the time. Gonna be a lot of API data.

Reply:
Removed a few calls. I was able to take in some data from the event itself rather than calling the API